languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml

```diff
@@ -58391,6 +58395,7 @@ USA
                     </token>
                 </marker>
             </pattern>
+            <!-- shorten -->
             <message suppress_misspelled="yes">Possible agreement error. The noun '<match no="3"/>' seems to be countable, so consider using: <suggestion><match no="3" postag="NNP?S" postag_regexp="yes"/></suggestion>.</message>
```

Comment:
Agreed. "Possible agreement error" is enough hedging. The suggestion can go outside the message.